rsc/extra/check_makefile.sh

```diff
@@ -10,7 +10,7 @@ git_repo=`git rev-parse --show-toplevel`
 local_install_dir=$1
 
 # Cd into the source directory
```

Comment:
Already in project root directory, this command is maybe not needed anymore. 
We can let it here, just to be sure and change comment
```suggestion
# Cd into the root project directory
```